cypress/e2e/template-starter/channels.cy.ts

```diff
@@ -10,7 +10,8 @@ describe('Channels', () => {
       cy.findByText('Channels list').should('exist');
     });
     cy.findByText('Store Berlin').should('exist');
-    cy.findByText('Store Munich').should('exist');
+    cy.findByText('Store Munich').should('exist').click();
+    cy.findByText('Store Munich - Template starter - mc-e2e-app-kit-01 - Merchant Center').should('exist');
```

Comment:
Did you check using `cy.title`?  https://docs.cypress.io/api/commands/title#Examples

```suggestion
    cy.title().should('eq', 'Store Munich - Template starter - mc-e2e-app-kit-01 - Merchant Center');
```

Reply:
Updated here: https://github.com/commercetools/merchant-center-application-kit/pull/2804/commits/5601a8028f7a07257709dbf46fc10716bcbaf628
Thank you.